### PR TITLE
Link prism theme at layout-core render

### PR DESCRIPTION
**Pull request by m4dz**

### CODE REVIEW

**Comment by georges-gomes** [comment]
I'm wondering if a `import prismcss from 'prismjs@1.25.0/themes/prism-tomorrow.css?raw'` would work and would be js embedded.

**Comment by gqio** [comment]
> I'm wondering if a `import prismcss from 'prismjs/themes/prism-tomorrow.css?raw'` would work and would be js embedded.

You can check with a call to `/pack_bundle`

File: mdx-layout-core/src/CoreLayout.tsx [hunk 1]
```diff
@@ -42,6 +40,12 @@ export const CoreLayout = ({
     return () => mediaMatch.removeEventListener('change', handler);
   });
 
+  const link = document.createElement('link');
```
Comment (gqio) [inline mdx-layout-core/src/CoreLayout.tsx:43]: Hummmm .....react  does support link inside JSX ?

**Comment by m4dz** [comment]
OK, moved to the `?raw` import of Vite then injecting prism styles right in the `<main>` element, thanks for the suggestion @georges-gomes !

**Review by gqio — APPROVED** [review]

**Pull request merged**